Operators.ipynb

```diff
@@ -0,0 +1,361 @@
+     "text": [
+      "a / b = 3.3333333333333335\n",
+      "a // b = 3\n",
+      "a % b = 5\n"
+     ]
+    }
+   ],
+   "source": [
+    "a = 50\n",
+    "b = 15\n",
+    "\n",
+    "print(\"a / b =\", a / b)\n",
+    "print(\"a // b =\", a // b)\n",
+    "print(\"a % b =\", a % b)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2dbd0cb9-ca39-482c-b7e1-63bb00f02594",
+   "metadata": {},
+   "source": [
+    "## Bitwise Operators\n",
+    "\n",
+    "They are used to operate on binary numbers and perform the bit-by-bit operations\n",
+    "\n",
+    "| Operator | Name | Example |\n",
+    "| ---- | ---- | ---- |\n",
+    "| & | Bitwise AND | x & y |\n",
+    "| \\| | Bitwise OR | x | y |\n",
+    "| ~ | Bitwise NOT | ~x |\n",
+    "| ^ | Bitwise XOR | x ^ y |\n",
+    "| >> | Bitwise right shift | x >> y |\n",
+    "| << | Bitwise left shift | x << y |"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "id": "bcc05fff-72bf-40f8-9b5a-ef1a9906e294",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "a & b = 0\n",
+      "a | b = 7\n",
+      "~b = -4\n",
+      "b >> 2 = 1\n",
+      "b << 2 = 16\n"
+     ]
+    }
+   ],
+   "source": [
+    "a = 3\n",
+    "b = 4\n",
+    "\n",
+    "print(\"a & b =\", a & b)\n",
+    "print(\"a | b =\", a | b)\n",
+    "print(\"~b =\", ~a)\n",
+    "print(\"b >> 2 =\", b >> 2)\n",
+    "print(\"b << 2 =\", b << 2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5107ea69-c37d-49f3-bf54-98a21a4748f2",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Assignment Operators\n",
+    "\n",
+    "They are used to assign values to variables:\n",
+    "    \n",
+    "| Operator | Example | Same As |\n",
+    "| ---- | ---- | ---- |\n",
+    "| = | y = 5 |  |\n",
+    "| (op)= | x += 3 | x = x + 3 |"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 75,
+   "id": "95cb8fa4-acc1-4acb-998c-c7373a12739d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "a += b = 14\n",
+      "a //= b = 2\n",
+      "a **= b = 10000\n",
+      "a |= b = 14\n"
+     ]
+    }
+   ],
+   "source": [
+    "b = 4\n",
+    "\n",
+    "a = 10\n",
+    "a += b\n",
+    "print(\"a += b =\", a)\n",
+    "\n",
+    "a = 10\n",
+    "a //= b\n",
+    "print(\"a //= b =\", a)\n",
+    "\n",
+    "a = 10\n",
+    "a **= b\n",
+    "print(\"a **= b =\", a)\n",
+    "\n",
+    "a = 10\n",
+    "a |= b\n",
+    "print(\"a |= b =\", a)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "924743aa-49f5-4b9c-972c-c77c15147f87",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Comparison Operators\n",
+    "\n",
+    "They are used to compare two values:\n",
+    "\n",
+    "| Operator | Name | Example |\n",
+    "| ---- | ---- | ---- |\n",
+    "| == | Equal to | x == y | \n",
```

Comment:
Equals

Reply:
@jmbuzano Seguro? también se puede agregar 'is' al inicio.

Reply:
NA